packages/snaps-execution-environments/src/common/BaseSnapExecutor.test.browser.ts

```diff
@@ -20,10 +21,7 @@ import { testEndowmentHardening } from './test-utils/hardening';
 import 'ses';
 
 describe('BaseSnapExecutor', () => {
-  before(() => {
```

Comment:
Why isn't this needed anymore?

Reply:
I'm not sure why it was necessary in the first place anymore. It works without it now.